wasm middleware: change path property to url (#3359)
dapr/docs: daprdocs/content/en/reference/components-reference/supported-middleware/middleware-wasm.md

pre-compiled into the `daprd` binary. Dapr embeds [wazero](https://wazero.io)
to accomplish this without CGO.

> Wasm modules are loaded from a URL. For example, a URL "file://rewrite.wasm"

Reviewer: ```suggestion
Wasm binaries are loaded from a URL. For example, the URL `file://rewrite.wasm`
```

> loads "rewrite.wasm" from the current directory of the process. On Kubernetes,

Reviewer: ```suggestion
loads `rewrite.wasm` from the current directory of the process. On Kubernetes,
```

> see [mounting volumes to the Dapr sidecar]({{< ref kubernetes-volume-mounts.md >}})

Reviewer: ```suggestion
see [How to: Mount Pod volumes to the Dapr sidecar]({{< ref kubernetes-volume-mounts.md >}})
```

  type: middleware.http.wasm
  version: v1
  metadata:
  - name: url
    value: "file://router.wasm"
```

## Spec metadata fields

Minimally, a user must specify a Wasm binary implements the [http-handler](https://http-wasm.io/http-handler/).
How to compile this is described later.

| Field | Details                                                        | Required | Example        |
|-------|----------------------------------------------------------------|----------|----------------|

> | url   | The URL of the resource including the Wasm binary to instantiate. The supported schemes include "file://". The path of a "file://" URL is relative to the Dapr process unless it begins with '/'. | true     | "file://hello.wasm" |

Reviewer: ```suggestion
| url   | The URL of the resource including the Wasm binary to instantiate. The supported schemes include `file://`. The path of a `file://` URL is relative to the Dapr process unless it begins with `/`. | true     | `file://hello.wasm` |
```

```

If using TinyGo, compile as shown below and set the spec metadata field named

> "url" to the location of the output (ex "file://router.wasm"):

Reviewer: ```suggestion
"url" to the location of the output (ex "file://router.wasm"):
```
```suggestion
"url" to the location of the output (for example, `file://router.wasm`):
```

```

If using TinyGo, compile as shown below and set the spec metadata field named

> "url" to the location of the output (ex "file://example.wasm"):

Reviewer: ```suggestion
"url" to the location of the output (for example, `file://example.wasm`):
```
